spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -52,6 +52,42 @@
     end
   end
 
+  context "a ActiveModel that's validate a delegated attribute to a ActiveModel" do
+    before do
+      parent = define_active_model_class('Parent', accessors: [:children]).new
+      @model = define_active_model_class('Example', accessors: [:parent]) do
+        delegate :children, :children=, to: :parent
+        validates_presence_of :children
+      end.new(parent: parent)
```

Comment:
Can you put the `.new(parent: parent)` on its own line? This looks a little funny.